juniper-junos/info.json

```diff
@@ -0,0 +1,367 @@
+{
+    "name": "juniper-junos",
+    "version": "1.0.0",
+    "label": "Juniper JunOS",
+    "description": "Provides JunOS REST API Integration covering Juniper MX, PTX, QFX, T and SRX Series platforms",
+    "publisher": "",
```

Comment:
```suggestion
    "publisher": "Fortinet",
```

Reply:
Fortinet CSE is what has been agreed with Pooja, check please.